omod-common/src/main/java/org/openmrs/module/webservices/rest/web/resource/api/Deletable.java

```diff
@@ -28,4 +28,16 @@
 	 */
 	void delete(String uuid, String reason, RequestContext context) throws ResponseException;
 	
+	/**
+	 * Undeletes the specified resource, which in the OpenMRS context means either unvoiding or
+	 * unretiring it
+	 * 
+	 * @param uuid
```

Comment:
Did you just forget to document these params?

Reply:
You mean `uuid` and `context` ? If so what in their documentation is missing?

Reply:
That was quick, the description is missing. But This is how the other params I have seen in this module being documented.

Reply:
Did you do some googling about javadoc?

Reply:
Definitely yes, and in this case I had to remind myself. Good convention is to include the `@params`'s description. Gona do just that. Thank you @dkayiwa .